client/state/stats/lists/utils.js

```diff
@@ -150,6 +154,58 @@ function parseOrderDeltas( payload ) {
  * a null value
  * @return {array} - Array of data objects
  */
+function parseOrdersChartData( payload, nullAttributes = [] ) {
+	if ( ! payload || ! payload.data ) {
+		return [];
+	}
+
+	return payload.data.map( function( record ) {
+		// Initialize data
+		const dataRecord = nullAttributes.reduce( ( memo, attribute ) => {
```

Comment:
You can remove the `nullAttributes` logic. That was there to make `parseChartData` compatible with the previous usage.